core/core.go

```diff
@@ -114,6 +114,11 @@ type Config struct {
 	//    }
 	//
 	CustomContractAddresses string `envvar:"CUSTOM_CONTRACT_ADDRESSES" default:""`
+	// MaxOrdersInStorage is the maximum number of orders that Mesh will keep in
+	// storage. As the number of orders in storage grows, Mesh will begin
+	// enforcing a limit on maximum expiration time for incoming orders and remove
+	// any orders with an expiration time too far in the future.
+	MaxOrdersInStorage int `envvar:"MAX_ORDERS_IN_STORAGE" default:"10000"`
```

Comment:
How many orders does GUDecks have? Think we might need to default this to a higher number. Thoughts?

Reply:
Just changed it from 10k to 100k. This keeps us [just under 1GB](https://www.wolframalpha.com/input/?i=8192+bytes+*+100000) (with some room for indexes) in the worst case scenario where every order is the max size.

GUDecks told us they have about 5k orders now so this leaves room for them to grow in the future. Also any NFT marketplaces with a larger number of orders can always configure their nodes to have a higher limit.